doc/introduction/chemistry.rst

```diff
@@ -21,38 +21,38 @@ Eigensolver (VQE) algorithm.
 Building the electronic Hamiltonian
 -----------------------------------
 
-The ``qchem`` module provides access to a driver function :func:`~.generate_hamiltonian`
+The ``qchem`` module provides access to a driver function :func:`~.molecular_hamiltonian`
 to generate the electronic Hamiltonian in a single call. For example,
 
 .. code-block:: python
 
-    h, nr_qubits = qml.qchem.generate_hamiltonian(
+    h, n_qubits = qml.qchem.molecular_hamiltonian(
         name='h2',
         geo_file='h2.xyz',
         charge=0,
-        multiplicity=1,
-        basis_set='sto-3g',
-        n_active_electrons=2,
-        n_active_orbitals=2
+        mult=1,
+        basis='sto-3g',
+        nact_els=2,
+        nact_orbs=2
     )
 
 where:
 
 * ``h`` is the qubit Hamiltonian of the molecule represented as a PennyLane Hamiltonian, and
 
-* ``nr_qubits`` is the number of qubits operators needed to represent it.
+* ``n_qubits`` is the number of qubits operators needed to represent it.
```

Comment:
"qubit operators" ?

Reply:
I personally prefer `nr_qubits` since it naturally reads as "number of qubits". My preference would be to just use `qubits`. Also I agree with Soran, it's awkward to say qubit operators. Why not just say "the number of qubits needed to represent it"?